.github/workflows/maven-publish.yml

```diff
@@ -6,7 +6,7 @@ name: Maven Package
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Please keep master

Reply:
After check , i saw 
I was not change any things into this file (.github/workflows/maven-publish.yml)  for this task .